debian/ovn-central.template

```diff
@@ -2,12 +2,13 @@
 
 # OVN_CTL_OPTS: Extra options to pass to ovs-ctl.  This is, for example,
 # a suitable place to specify --ovn-northd-wrapper=valgrind.
-OVN_CTL_OPTS="--db-nb-port=6641 \
+HOST_IP="0.0.0.0"
```

Comment:
If you want to set this from ansible, don't you want this to be:

HOST_IP=${HOST_IP:-0.0.0.0}